pkg/api/config/config.go

```diff
@@ -102,6 +102,7 @@ type SchedulerConfig struct {
 }
 
 type LDAPConfig struct {
+	LDAPConfigFile     string
```

Comment:
LDAPCredentialsFile only pls, no need to put all config in that file.

Reply:
To clarify, credentials being "BindDN" and "BindPassword".

Reply:
yup, and that can be in JSON